README.md

```diff
@@ -152,6 +152,7 @@ With values used for `--format` option:
 - [x] Golang coverage format - `golang`
 - [x] Coveralls JSON format - `coveralls`
 - [x] Pytest-Cov ([:test_tube: beta](#pytest-cov-test_tube-beta)) - `python`
+- [x] Clover XML - `phpunit`
```

Comment:
Is this correct or should this be `clover` ? I didn't see `phpunit` mentioned anywhere in PR #99 ?

Reply:
I've never seen a binary named "clover".

Reply:
It's not about the binary. This section is about what name to use when passing the `format` parameter. As far as I can see, that should be `clover`, not `phpunit`, though I may well be wrong ?

If it turns out I'm right, I'd suggest updating the entry to "[x] Clover XML as available via PHPUnit - `clover`"

Reply:
I think you're right, @jrfnl. Thanks for catching that, and for the suggestion.

Reply:
@afinetooth PR #108 is already open to fix this.